cms/forms/wizards.py

```diff
@@ -114,7 +114,7 @@ def __init__(self, instance=None, *args, **kwargs):
         self.instance = instance
         super(BaseCMSPageForm, self).__init__(*args, **kwargs)
 
-        if self.page:
+        if getattr(self, 'page', None):
```

Comment:
Please don't use getattr unless it's necessary.
This attribute is (should) always be present (even if none).